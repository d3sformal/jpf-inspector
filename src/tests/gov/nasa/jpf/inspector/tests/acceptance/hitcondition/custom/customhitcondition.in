cr bp line(7)
cr bp line(not a number)
cr bp line(bad, number, of, arguments)
run && wait
run && wait
run && wait