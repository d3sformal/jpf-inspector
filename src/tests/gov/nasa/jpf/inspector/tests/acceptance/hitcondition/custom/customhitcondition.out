ERR: The only argument must be a line number.
ERR: There must be exactly one argument.
INFO: JPF created and connected, SuT is started
INFO: Breakpoint hit: [1 :  state=ENABLED hits=1 hitsTotal=1 line(7)]
	These are details of the example custom hit condition.
INFO: Breakpoint hit: [1 :  state=ENABLED hits=2 hitsTotal=2 line(7)]
	These are details of the example custom hit condition.
INFO: JPF Terminating